pkg/oci/utils.go

```diff
@@ -37,6 +37,34 @@ var (
 
 	// BundlePathKey is the annotation key to fetch the OCI configuration file path.
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
+
+	// ContainerTypeKey is the annotation key to fetch container type.
+	ContainerTypeKey = "com.github.containers.virtcontainers.pkg.oci.container_type"
+
+	// CRIOContainerTypeKey is the annotation key to fetch CRIO container type.
+	CRIOContainerTypeKey = "crio/container_type"
```

Comment:
Hmm, it would have been nicer if they had created a `const` but how about we add those links into the code? I can see this breaking rather easily (through no fault of ours), but atleast if we add a reference, there is some hope of finding the new values when they change them ;)